client/blocks/upsell-nudge/docs/example.jsx

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import UpsellNudge from 'blocks/upsell-nudge';
+
+const UpsellNudgeExample = ( { translate } ) => (
+	<>
+		<UpsellNudge
+			href="#"
+			callToAction={ translate( 'Upgrade' ) }
```

Comment:
We don't need to localize components on the DevDocs, but it is not harmless. Feel free to either leave as-is or remove the localization before merging.